src/components/GettingStartedDropdown/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { ChevronDownIcon } from "@chakra-ui/icons";
+import {
+  forwardRef,
+  Button,
+  Menu,
+  MenuButton,
+  MenuGroup,
+  MenuItem,
+  MenuList,
+  PropsOf,
+} from "@chakra-ui/react";
+import { GETTING_STARTED, ILink } from "../../constants/links";
+
+function ExternalLink({ display, url }: ILink): JSX.Element {
+  return (
+    <MenuItem>
+      <a href={url}>{display}</a>
+    </MenuItem>
+  );
+}
+
+const GettingStartedButton = forwardRef(
+  (props: PropsOf<typeof Button>, ref) => {
+    return <Button bg="white" ref={ref} size="md" {...props} />;
+  }
+);
+
+GettingStartedButton.displayName = "GettingStartedButton";
+
+export default function GettingStartedDropdown(): JSX.Element {
+  return (
+    <Menu>
+      <MenuButton
+        as={GettingStartedButton}
+        rightIcon={<ChevronDownIcon color="black.600" h={6} w={6} />}
+      >
```

Comment:
```suggestion
      <MenuButton
        as={Button}
        bg="white"
        size="md"
        rightIcon={<ChevronDownIcon color="black.600" h={6} w={6} />}
      >
```

I know I did the same thing as this on the Dependency Dropdown, but it turns out it wasn't necessary at all 😅

Reply:
Ah is this cuz header is used outside the routing?

Reply:
We resolved this conversation offline, but the gist is that we can just pass in the chakra `<Button />` and `<MenuButton />` will infer the correct prop typings, so defining a separate forwardRef component to pre-define some props isn't needed